detox/package.json

```diff
@@ -11,6 +11,7 @@
     "axios": "0.21.1",
     "babel-jest": "26.6.3",
     "babel-plugin-module-resolver": "4.1.0",
+    "client-oauth2": "4.3.3",
```

Comment:
Are we going to use the upstream package or the fork from Daniel? In Cypress is like [this](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/package.json#L12).

Reply:
@saturninoabril I'm not sure. That would be a question for @larkox .

@larkox Please let me know if you need me to use the fork you created.

Reply:
Done